lib/load/merge-configs.js

```diff
@@ -4,7 +4,7 @@ const merge = require('lodash').mergeWith;
 
 module.exports = function(configs) {
     if (!configs || !configs.length) return {};
-    return merge({}, ...configs, (a, b) => {
+    return merge({}, configs, (a, b) => {
```

Comment:
meh, i knew i shouldn't have been lazy and submitted that PR with that commit message. 

It throws out a gulp error when you try to load a config file. I'll get an example typed up and submit it here tomorrow for potential consideration